packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -193,7 +193,7 @@ export const runFrameworkGenerator = async (ctx: C3Context, args: string[]) => {
 
 	endSection(
 		`Continue with ${ctx.framework?.config.displayName}`,
-		`via \`${quoteShellArgs(cmd)}\``
+		`via \`${cmd.join(" ")}\``
```

Comment:
I guess this is to get rid of the `\` in front of the `:`? cc @dario-piotrowicz

Reply:
If this is annoying enough, then we should consider whether using `shell-quote` is a good idea at all. Perhaps just hand code our own helper?

Reply:
I've made a `formatCommand` helper for now that we can replace `quoteShellArgs` with in places where we're just displaying the command to the user.